src/constants.ts

```diff
@@ -19,6 +19,7 @@
 const headerNames = {
   hops: 'X-AR-IO-Hops',
   origin: 'X-AR-IO-Origin',
+  nodeRelease: 'X-AR-IO-Node-Release',
```

Comment:
I wonder if we should change this to `X-AR-IO-Origin-Node-Release`. Otherwise I think we may find it confusing in the future (is it this node or the origin?).